pom.xml

```diff
@@ -45,9 +45,9 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-cosmosdb</artifactId>
-            <version>2.2.0</version>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-cosmos</artifactId>
+            <version>4.0.0-preview.1</version>
```

Comment:
Change to latest version.

Reply:
The method is an internal method.
![image](https://user-images.githubusercontent.com/64007802/86452717-4b343480-bd4f-11ea-9e7f-d870444cb812.png)